client/my-sites/stats/wordads/style.scss

```diff
@@ -2,4 +2,15 @@
 	.wordads .stats-tabs .stats-tab {
 		width: 33.33%;
 	}
+
+	.earnings_history td:hover,
+	tbody tr:hover td,
```

Comment:
Did you want to add `.earnings_history` before tbody?

Reply:
Nice catch, thanks. Going to fix this up 🙌